README.md

```diff
@@ -148,6 +148,7 @@ For local development and familiarizing yourself with Storm/Mesos, please see th
 * `mesos.master.failover.timeout.secs`: Framework failover timeout in second. Defaults to "24*7*3600".
 * `mesos.allowed.hosts`: Allowed hosts to run topology, which takes hostname list as a white list.
 * `mesos.disallowed.hosts`: Disallowed hosts to run topology, which takes hostname list as a back list.
+* `mesos.constraints`: Constrain slave nodes which bootstrap supervisor based on mesos-attributes.  
```

Comment:
Oh... I didn't make it clear enough. 
OK, I will try to add new section to README about attribute constraints